packages/terra-site/package.json

```diff
@@ -88,25 +88,25 @@
   "devDependencies": {
     "autoprefixer": "^6.7.7",
     "babel-cli": "^6.24.1",
-    "babel-loader": "^7.0.0",
+    "babel-loader": "^7.1.2",
     "babel-polyfill": "^6.23.0",
-    "clean-webpack-plugin": "^0.1.16",
-    "css-loader": "^0.28.0",
-    "extract-text-webpack-plugin": "^2.1.0",
-    "file-loader": "^0.11.1",
+    "clean-webpack-plugin": "^0.1.17",
+    "css-loader": "^0.28.7",
+    "extract-text-webpack-plugin": "^3.0.1",
+    "file-loader": "^1.1.5",
     "gh-pages": "^0.12.0",
-    "html-webpack-plugin": "^2.28.0",
-    "json-loader": "^0.5.4",
+    "html-webpack-plugin": "^2.30.0",
+    "json-loader": "^0.5.7",
     "node-sass": "^4.5.2",
     "postcss-custom-properties": "^6.0.1",
-    "postcss-loader": "^1.3.3",
-    "postcss-rtl": "^0.5.10",
+    "postcss-loader": "^2.0.6",
+    "postcss-rtl": "^1.1.2",
     "raw-loader": "^0.5.1",
     "react": "^15.4.2",
     "react-dom": "^15.4.2",
-    "sass-loader": "^6.0.3",
-    "style-loader": "^0.16.1",
-    "webpack": "^2.6.1",
-    "webpack-dev-server": "^2.4.5"
+    "sass-loader": "^6.0.6",
+    "style-loader": "^0.19.0",
+    "webpack": "^3.6.0",
```

Comment:
It appears there are nightwatch tests failures after updating webpack. We'll want to get this resolved, feel free to reach out to @emilyrohrbough and @ryanthemanuel as there the most familiar with this.

Reply:
This PR should address those failures https://github.com/cerner/terra-toolkit/pull/14/files

Reply:
Gonna clear out travis CI's node_modules cache and rerun the build. This *should* pick up the toolkit v2.1.0 release

Reply:
oh, i explicitly updated to it

Reply:
Might need to update the version in terra-site as well: https://github.com/cerner/terra-core/blob/master/packages/terra-site/package.json#L109-L110

Reply:
i don't think it actually uses that version

Reply:
Ok once tests pass, lets get this merged down.